device-registry/src/main/scala/org/genivi/sota/device_registry/DevicePackagesResource.scala

```diff
@@ -49,6 +57,9 @@ class DevicePackagesResource(namespaceExtractor: Directive1[AuthedNamespaceScope
           }
         }
       }
+    } ~
+    (scope.get & pathPrefix("package") & extractPackageId & path("stats")) { pkg =>
```

Comment:
Let's not have 'package' path, keep it about devices. Eg 'devices_count' with parameter 'withPackage' etc.